package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "factorio_blueprint_editor",
+  "version": "1.0.0",
+  "description": "Online Editor for Elxeno's Factorio Blueprint Editor",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/.bin/eslint js/**/*.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/camerongillette/factorio_blueprint_editor.git"
+  },
+  "keywords": [
+    "Factorio",
+    "Blueprints",
+    "Tool"
+  ],
+  "author": "camerongillette, Elxeno",
```

Comment:
I think this is the right author. should we change it?

Reply:
It probably doesn't matter, but I think npm wants something a little different here, see https://docs.npmjs.com/files/package.json#people-fields-author-contributors

```js
 "author": { "name": "Elxeno" },
 "contributors": [
    {"name": "camerongillette"}
  ]
```